packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -399,13 +448,16 @@ export default compose( [
 			selectedLastClientId: getLastMultiSelectedBlockClientId(),
 			hasMultiSelection: hasMultiSelection(),
 			blocks: getBlockOrder(),
+			isNavigationMode: isNavigationMode(),
 		};
 	} ),
 	withDispatch( ( dispatch ) => {
-		const { multiSelect, selectBlock } = dispatch( 'core/block-editor' );
+		const { multiSelect, selectBlock, toggleNavigationMode, clearSelectedBlock } = dispatch( 'core/block-editor' );
 		return {
 			onMultiSelect: multiSelect,
 			onSelectBlock: selectBlock,
+			disableNavigationMode: () => toggleNavigationMode( false ),
```

Comment:
All these different verbs confuse me: `toggle`, `disable`, `switchTo`. Could it be unified? Also, to me, `disable` sounds like it will be a mode that is no longer available.

Atm you have: switchToEditMode => disableNavigationMode => toggleNavigationMode.